tests/integration/pni/articles.spec.js

```diff
@@ -0,0 +1,50 @@
+const { test, expect } = require("@playwright/test");
+const waitForImagesToLoad = require("../../wait-for-images.js");
+
+test.describe("PNI articles", () => {
+  test(`"load more" button`, async ({ page }) => {
+    page.on(`console`, console.log);
+    await page.goto(`http://localhost:8000/en/privacynotincluded/articles`);
+    await page.locator(`body.react-loaded`);
+    await waitForImagesToLoad(page);
+
+    const articleList = page.locator(`#items-list`);
+    expect(await articleList.count()).toEqual(1);
+
+    const articles = articleList.locator(`li`);
+    const numArticles = articles.count();
+    expect(await numArticles).toBeGreaterThan(1);
+
+    const loadMoreButton = page.locator(
+      `#load-more button[data-hx-get][data-hx-push-url]`,
+    );
+    expect(await loadMoreButton.count()).toBe(1);
+    expect(await loadMoreButton.isVisible()).toBe(true);
+
+    let getUrl = await loadMoreButton.getAttribute(`data-hx-get`);
+    let pushUrl = await loadMoreButton.getAttribute(`data-hx-push-url`);
+    // these need to be full URL for the following assertion to work
+    getUrl = `http://localhost:8000${getUrl}`;
```

Comment:
hard coded `http://localhost:8000` here but this will be fixed as part of the https://github.com/MozillaFoundation/foundation.mozilla.org/pull/12132